tests/udp-server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-alpine as builder
```

Comment:
@soneillf5 minor one but  can we one readme and dockerfile for both tcp and udp server and ask the user to pass port as var in docker-compose?

Reply:
fixed